app/util/pre_run/environment_compliance_check.py

```diff
@@ -60,6 +72,21 @@ def validate_application_config(processors, app_name_upper, app_settings, min_de
                 f"To skip environment compliance check set `environment_compliance_check` variable to False in your {app_name_upper.lower()}.yml file.")
 
 
+def validate_chromedriver_version(app_name, app_settings):
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless")
+    driver = webdriver.Chrome(options=options)
+    current_chrome_version = version.parse(
+        driver.capabilities['browserVersion'])
+    current_chromedriver_version = version.parse(
+        app_settings.chromedriver_version)
+    if current_chromedriver_version != current_chrome_version:
```

Comment:
I suppose we need to compare only the major version, but not the full version.
E.g. My current latest Chrome is `122.0.6261.112`, but the latest chomedriver is `122.0.6261.111`

Reply:
Fixed